roxie/ccd/ccdactivities.cpp

```diff
@@ -3031,7 +3037,11 @@ class CRoxieIndexReadActivity : public CRoxieIndexActivity, implements IIndexRea
         {
             if (doTrace(traceSmartStepping, TraceFlags::Max) && !aborted)
             {
-                logctx.CTXLOG("Indexread returning result set %d rows from %d seeks, %d scans, %d skips", processed-processedBefore, tlk->querySeeks(), tlk->queryScans(), tlk->querySkips());
+                const CRuntimeStatisticCollection & stats = logctx.queryStats();
+                unsigned __int64 seeks = stats.getStatisticValue(StNumIndexSeeks);
+                unsigned __int64 scans = stats.getStatisticValue(StNumIndexScans);
+                unsigned __int64 skips = stats.getStatisticValue(StNumIndexSkips);
+                logctx.CTXLOG("Indexread returning result set %d rows from %" I64F"u seeks, %" I64F"u scans, %" I64F"u skips", processed-processedBefore, seeks, scans, skips);
```

Comment:
may not be trivial, but wondering if the 4 incidents of this type of code ( I think they're same with different messages?) could be commoned up.

Reply:
Possibly but would prefer to keep as it is to stop the scope of this jira from creeping.